frontend/src/scenes/retention/RetentionContainer.tsx

```diff
@@ -11,7 +11,7 @@ export function RetentionContainer({
 }): JSX.Element {
     return (
         <div className="retention-container">
-            <RetentionLineGraph inCardView={inCardView} inSharedMode={inSharedMode} />
+            {!inCardView && <RetentionLineGraph inCardView={inCardView} inSharedMode={inSharedMode} />}
```

Comment:
looks like we can remove the `inCardView` prop entirely now?

Reply:
Ofc

Reply:
and here toohttps://github.com/PostHog/posthog/blob/12a69c1965343dbfc0a932ee682d4672c11cf515/frontend/src/scenes/retention/RetentionLineGraph.tsx#L14

?? ☺️